patterns/base/a/a.yml

```diff
@@ -0,0 +1,9 @@
+name: Anchor Link
+description: Anchor Link
+twig: ./a.twig
+sass: ./a.scss
+category: base
+variables:
```

Comment:
Where did that come from? the current master has "data"
https://github.com/pattern-library/pattern-library/blob/master/patterns/base/h1/pattern.yml

On Fri, Apr 10, 2015 at 3:53 PM, Rebecca <notifications@github.com> wrote:

> In patterns/base/a/a.yml
> <https://github.com/pattern-library/pattern-library/pull/2#discussion_r28175071>
> :
>
> > @@ -0,0 +1,9 @@
> > +name: Anchor Link
> > +description: Anchor Link
> > +twig: ./a.twig
> > +sass: ./a.scss
> > +category: base
> > +variables:
>
> okay I was following the h1 yml file. Should that be data too?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pattern-library/pattern-library/pull/2/files#r28175071>
> .
>